core/rest/rest_api.json

```diff
@@ -5,7 +5,7 @@
         "description": "Interact with the enterprise blockchain through Hyperledger Fabric API",
         "version": "1.0.0"
     },
-    "host": "127.0.0.1:3000",
+    "host": "127.0.0.1:5000",
```

Comment:
I'm a little confused by this change, but its probably because I don't know Swagger.  Did we have something listening on port 3000?  I was under the impression we were listening on port 5000 but Vagrant was simply remapping things.